src/mudlet-lua/lua/geyser/GeyserVBox.lua

```diff
@@ -27,7 +27,13 @@ end
 -- Called when a new element is added
 function Geyser.VBox:organize()
   self.parent:reposition()
-
+  -- Workaround for issue with width/height being 0 at creation
+  if self:get_width() == 0 then
+    self:resize("0.9px", nil)
+  end
+  if self:get_height() == 0 then
+    self:resize(nil, "0.9px")
+  end
   local window_height = (self:calculate_dynamic_window_size().height / self.get_height()) * 100
```

Comment:
why is this one self.get_height instead of self:get_height? Just noticed it

Reply:
probably just a typo/copy past fail :wink: